### PR TITLE
feat(#91): Adds Diferencia section

### PR DESCRIPTION
**Pull request by lordofthejars**
This PR fixes https://github.com/redhat-developer-demos/istio-tutorial/pull/46 PR (so it can be closed now)

Issue: #91

### CODE REVIEW

File: documentation/modules/advanced/pages/diferencia.adoc [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+kubectl create -f link:{github-repo}/{diferencia-dir}/recommendation-master-svc.yml[diferencia/recommendation-master-svc.yml] -n tutorial
+kubectl create -f link:{github-repo}/{diferencia-dir}/recommendation-candidate-svc.yml[diferencia/recommendation-candidate-svc.yml] -n tutorial
+----
+
+[#deploying-diferencia]
+=== Deploying Diferencia
+
+Next thing is to deploy Diferencia.
+
+[source, bash]
```
Comment (rafabene) [inline documentation/modules/advanced/pages/diferencia.adoc]: It misses the [source, bash,subs="+macros,+attributes"] so it is not rendering properly.

File: documentation/modules/advanced/pages/diferencia.adoc [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+----
+
+[#shadowing-traffic]
+=== Shadowing Traffic
+
+The last thing is to just start shadowing traffic between `recommendation:v1` and diferencia.
+
+[source, bash,subs="+macros,+attributes"]
+----
+istioctl create -f link:{github-repo}/{diferencia-dir}/virtual-service-recommendation-v1-diferencia-mirror-v2.yml[diferencia/virtual-service-recommendation-v1-diferencia-mirror-v2.yml] -n tutorial
```
Comment (rafabene) [inline documentation/modules/advanced/pages/diferencia.adoc]: It misses the destination rule file.

Comment (rafabene) [reply documentation/modules/advanced/pages/diferencia.adoc]: I couldn't proceed because it fails without the destination rule

Comment (lordofthejars) [reply documentation/modules/advanced/pages/diferencia.adoc]: The file is there probably the problem is that I am assuming user is in a directory which he isn't there. Let me review.

Comment (lordofthejars) [reply documentation/modules/advanced/pages/diferencia.adoc]: In theory it is correct. I have tried and it finds it, can you do a `pwd` and check you are in the root of `istio-tutorial?`

**Comment by rafabene** [comment]
Can you also rebase your commit so It can have the conflicts sorted out?

**Comment by lordofthejars** [comment]
It seems that now there are no conflicts.

**Comment by rafabene** [comment]
It is being mentioned by Github. I had a conflict in the nav.adoc file.
screenshot 2018-12-12 15 04 28

File: documentation/modules/advanced/pages/diferencia.adoc (hunk 2 above)
Comment (rafabene) [reply documentation/modules/advanced/pages/diferencia.adoc]: The destinatio-rule is not mentioned in the documentation. Only the virtual-service.

**Comment by lordofthejars** [comment]
but not in mine hahahaha it says that can be merged

File: documentation/modules/advanced/pages/diferencia.adoc (hunk 2 above)
Comment (lordofthejars) [reply documentation/modules/advanced/pages/diferencia.adoc]: true, adding it.

**Pull request merged**